nrf_rpc: Doc review from oyvindronningstad (#1)
doki-nordic/nrfxlib: nrf_rpc/doc/usage.rst

Encoders encode commands and events into serialized packets.
Creating an encoder is similar for all packet types.
The first step is the allocation of a buffer using :c:macro:`NRF_RPC_ALLOC`.
After that, you can encode parameters directly into the buffer or use the `TinyCBOR`_ library.
In the last step, the packet is sent using one of the sending functions: :cpp:func:`nrf_rpc_cmd`, :cpp:func:`nrf_rpc_cbor_evt`, or similar.

> After sending the command a response is received, so it has to be parsed.

Reviewer: ```suggestion
After sending the command, a response is received, so it must be parsed.
```

	}

The above code uses the ``remote_inc_rsp`` function to parse the response.

> The following code shows how this function may look.

Reviewer: ```suggestion
The following code shows how this function might look.
```
